nknd.go

```diff
@@ -196,18 +196,24 @@ func nknMain(c *cli.Context) error {
 		}
 	}
 
+	// init web service
+	dashboard.Init(nil, wallet, nil)
+
+	// start JsonRPC
+	rpcServer := httpjson.NewServer(nil, wallet)
+	go rpcServer.Start()
+
 	// initialize ledger
 	err = InitLedger(account)
 	if err != nil {
 		return fmt.Errorf("chain.initialization error: %v", err)
 	}
 
+	rpcServer.SetLedgerStatus(true)
```

Comment:
It's better to put this line after defer below so if this function call fails, the store will still be closed

Reply:
fixed